src/riak_kv_ts_svc.erl

```diff
@@ -136,7 +136,11 @@ process(M = #tslistkeysreq{table = Table}, State) ->
 process(M = #tscoveragereq{table = Table}, State) ->
     check_table_and_call(Table, fun sub_tscoveragereq/4, M, State);
 
-%% this is tsqueryreq, subdivided per query type in its SQL
+%% The following heads of `process` are all, in terms of protobuffer
```

Comment:
Let's do the Erlang edoc thing, even though this won't be an edoc, and use the proper backtick/apostrophe combination around process, as well as #tsqueryreq{} and other literals below.